pandas/core/computation/eval.py

```diff
@@ -206,19 +205,32 @@ def eval(expr, parser='pandas', engine=None, truediv=True,
         The number of prior stack frames to traverse and add to the current
         scope. Most users will **not** need to change this parameter.
     target : a target object for assignment, optional, default is None
```

Comment:
It really can be anything that supports string item-assignment, per the docstring I put.